site/access-control.xml

```diff
@@ -92,11 +92,17 @@ limitations under the License.
         <p>
           If you wish to allow the <code>guest</code> user to connect
           from a remote host, you should set the
-          <code>loopback_users</code> configuration item to
-          <code>[]</code>. A complete <code>rabbitmq.config</code>
+          <code>loopback_users</code> configuration to
+          <code>none</code>. A complete <a href="/configure.html">RabbitMQ config file</a>
           which does this would look like:
         </p>
         <pre class="sourcecode">
+loopback_users = none
+</pre>
+
+        Or, in the <a href="/configure.html#erlang-term-config-file">classic config file format</a> (<code>rabbitmq.config</code>):
```

Comment:
We still support this format, and it is still used in the advanced config. It would be odd to call something legacy and then use it for both backwards compatibility and advanced cases.